test/demo/constants.ts

```diff
@@ -1 +1,81 @@
+import {SERVICE_IDENTITY} from '../../src/client/internal';
+
+import {AppIdentity, NonDirectoryAppIdentity, DirectoryAppIdentity} from './utils/common';
+
 export const appStartupTime = 30 * 1000;
+
+export const testManagerIdentity: AppIdentity = {uuid: 'test-app', name: 'test-app'};
+
+export const resolverWindowIdentity: AppIdentity = {
```

Comment:
Could we not pull this from the provider somewhere? Wouldn't expect it to ever change, but seems dangerous defining it in two separate places.